hapic/context.py

```diff
@@ -106,8 +107,101 @@ def serve_directory(
         """
         raise NotImplementedError()
 
+    def handle_exception(
+        self,
+        exception_class: typing.Type[Exception],
+        http_code: int,
+    ) -> None:
+        raise NotImplementedError()
+
+    def handle_exceptions(
+        self,
+        exception_classes: typing.List[typing.Type[Exception]],
+        http_code: int,
+    ) -> None:
+        raise NotImplementedError()
+
+    def _add_exception_class_to_catch(
```

Comment:
private method in an interface is not common usage...

Reply:
Method deleted, this method should only exist in BaseContext you're right.